config/electron.config.js

```diff
@@ -1,8 +1,11 @@
+const ENVIRONMENT = require('../src/CONST/ENVIRONMENT');
+
 module.exports = {
     appId: 'com.expensifyreactnative.chat',
     productName: 'New Expensify',
     extraMetadata: {
         main: './desktop/main.js',
+        electronEnvironment: process.env.SHOULD_DEPLOY_PRODUCTION ? ENVIRONMENT.PRODUCTION : ENVIRONMENT.STAGING,
```

Comment:
This is where it still remains a bit weird for me. The `electronEnvironment` variable will only contain PROD or STAG. The development environment comes from `process.env.NODE_ENV`. Can't they all come from the same place? Like, can this code here be something like...

```
let env = process.env.NODE_ENV;
if (env !== 'development') {
    env = process.env.SHOULD_DEPLOY_PRODUCTION ? ENVIRONMENT.PRODUCTION : ENVIRONMENT.STAGING;
} else {
    env = ENVIRONMENT.DEVELOPMENT;
}

// ...
    electronEnvironment: env,
```

However, even in that code... if `process.env.NODE_ENV` is not `development`, then what is it? Maybe this logic can be cleaned up a little more.

The goal here would be that `getEnvironment()` (below) would only be looking at a single place to determine the environment.

Reply:
That sounds like a good idea.  As I understand it `process.env.NODE_ENV` would be undefined outside of development because:
```
function getEnvironment() {
    // If we are on dev, then the NODE_ENV environment variable will be present (set by the executing shell in start.js)
```

Reply:
That makes sense to me.